aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -289,6 +291,27 @@ Tensor _s_where(const Tensor& condition, const Tensor& self, const Tensor& other
   return ret;
 }
 
+void zero_numel_tensor_resize(Tensor& result, Tensor& result_indices, const Tensor& self, int64_t dim,
+                              bool keepdim) {
+  TORCH_CHECK(self.size(dim) != 0, "Expected reduction dim ", dim, " to be non-zero");
+
+  if (keepdim) {
+    auto sizes = ensure_nonempty_vec(self.sizes().vec());
+    sizes[dim] = 1;
+    result.resize_(sizes);
+  }
+  else {
+    std::vector<int64_t> sizes;
+    for (const auto d : c10::irange(self.dim())) {
+      if (d != dim) {
+        sizes.push_back(self.sizes()[d]);
+      }
+    }
+    result.resize_(sizes);
+  }
+  result_indices.resize_({});
```

Comment:
You sure about this? This is gonna give you a scalar (one element) tensor

Reply:
Changed.